catalog/db_batch_entry_read.go

```diff
@@ -0,0 +1,166 @@
+package catalog
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/treeverse/lakefs/db"
+)
+
+const (
+	MaxReadQueue        = 10
```

Comment:
I think all of these should be configurable parameters.  We will need different values at customer sites.

Reply:
You are right, I am just not sure where ti implement such low-level parameters